src/index.html

```diff
@@ -9,10 +9,45 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), class names should be kebab-case. The class name 'body' should be in kebab-case.